### ISSUE TITLE
DataError in Microsoft Edge

### ISSUE THREAD

**Issue by bennyn** [yathit/ydn-db#95]
Hello, by using YDN in my web application with Microsoft Edge I am getting a `DataError`. The JavaScript inspector says it comes from `key_range.js (265,3)`.

I am using:
- https://console.developers.google.com/m/cloudstorage/b/ydn-db-d1/o/1.3.6/ydn.db-i-core-e-qry-dev.js
- https://console.developers.google.com/m/cloudstorage/b/ydn-db-d1/o/1.3.6/ydn.db-i-core-e-qry-dev.js.map

When I run the same code in Chrome or Firefox, then it works.

**Comment by yathit** [comment]
Yes, it is known issue for over three years now. MS don't have time to fix, so I am.

**Comment by bennyn** [comment]
So can you tell me what's the problem in Edge? Why is it complaining? Maybe I can setup a different database schema to go work around this issue.

**Comment by yathit** [comment]
Yes, avoid multiEntry and compound index. See more on https://dev.yathit.com/ydn-db/doc/setup/limitation.html

**Comment by bennyn** [comment]
Ok, thanks for the fast reply. So with Edge I cannot use a schema like this:

```javascript
{
  name: 'name, timestamp',
  keyPath: [
    'name', 'timestamp'
  ]
}
```

But this is needed when I want to do queries like:
```javascript
db.from('users').where('name', '=', 'Homer').order('timestamp');
```

Is there another way of not using a compound index but being able to query by `name` and order by `timestamp`?

**Comment by mtycholaz** [comment]
There is support in Edge for basic IndexDb. They're considering adding more complex features. More info from Microsoft regarding this here.

https://wpdev.uservoice.com/forums/257854-microsoft-edge-developer/suggestions/8437039-full-feature-support-for-indexeddb
